avalon/nuke/pipeline.py

```diff
@@ -60,27 +53,27 @@ def containerise(node,
                  context,
                  loader=None,
                  data=None):
-    """Bundle `nodes` into an assembly and imprint it with metadata
+    """Bundle `node` into an assembly and imprint it with metadata
 
     Containerisation enables a tracking of version, author and origin
     for loaded assets.
 
     Arguments:
-        node (object): The node in Nuke to imprint as container,
-        usually a Reader.
+        node (obj): Nuke's node object to imprint as container,
+                    usually a Reader.
         name (str): Name of resulting assembly
         namespace (str): Namespace under which to host container
         context (dict): Asset information
         loader (str, optional): Name of node used to produce this container.
 
     Returns:
-        Node
+        node (obj): containerised nuke's node object
 
     """
 
     data_imprint = OrderedDict({
         "schema": "avalon-core:container-2.0",
-        "id": AVALON_CONTAINER_ID,
+        "id": "pyblish.avalon.container",
         "name": str(name),
```

Comment:
Is there any reason you need to revert this ? :O
It was from #352 , to avoid hard-coding container id everywhere. :)

Reply:
yes, this also makes sense ;)